src/pages/Likes/Likes.jsx

```diff
@@ -43,23 +44,25 @@ export default function Likes() {
   }, [fs, auth.currentUser.uid]);
 
   return (
-    <>
-      <ListGroup id="liked-recipe-list">
-        {recipes.map((recipe) => (
-          <Card id="liked-recipe-card" key={recipe.id}>
-            <Card.Img id="recipe-img" src={recipe.imgPath} />
-            <Card.Body>
-              <Card.Title>{recipe.name}</Card.Title>
-              <Card.Text className="recipe-description">
-                {recipe.description}
-              </Card.Text>
-              <Card.Text className="recipe-cook-time">
-                {recipe.cookingTime} min
-              </Card.Text>
-            </Card.Body>
-          </Card>
-        ))}
-      </ListGroup>
-    </>
+    <ListGroup id="liked-recipe-list">
+      <PlaceholderCard
+        cardTitle="No liked recipes"
+        cardBody="All you liked recipes appear here."
```

Comment:
maybe change this to "No liked recipes yet" and for sure change the second sentence to "All your liked recipes will appear here"